### PR TITLE
Feature/urgent orders

### PR DESCRIPTION
**Pull request by tamaragovorcin**

### CODE REVIEW

File: IntegrationWithPharmacies/Program.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public class Program
         public static string Environment = "Local";
         public static void Main(string[] args)
         {
-            CreateHostBuilderMessages(args).Build().Run();
+           // CreateHostBuilderMessages(args).Build().Run();
```
Comment (Nemanja0505) [inline IntegrationWithPharmacies/Program.cs:16]: Delete unnecessary comment .

**Review by Nemanja0505 — CHANGES_REQUESTED** [review]
PR cannot be approved, delete marked parts.

**Review by Nemanja0505 — APPROVED** [review]
PR is approved. Good Clean code practice has been followed.

File: IntegrationWithPharmacies/Services/UrgentOrderService.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        }
+        public Boolean SendOrderHttp(UrgentMedicineOrder order)
+        {
+            String medicineOrder = CreateOrder(order);
+            try
+            {
+                HttpService.SendUrgentOrder(medicineOrder);
+                return true;
+            }
+            catch (Exception e) { return false; }
```
Comment (trki293) [inline IntegrationWithPharmacies/Services/UrgentOrderService.cs:55]: Write this in form:
"try
            {
                HttpService.SendUrgentOrder(CreateOrder(order));
                return true;
            }
            catch (Exception e) { return false; }"

File: IntegrationWithPharmacies/Services/UrgentOrderService.cs [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        }
+        public Boolean SendOrderGrpc(UrgentMedicineOrder order)
+        {
+            String medicineOrder = CreateOrder(order);
+            try
+            {
+                string response = new ClientScheduledService().SendMessage(medicineOrder).Result;
+                return true;
+            }
+            catch (Exception e) { return false; }
```
Comment (trki293) [inline IntegrationWithPharmacies/Services/UrgentOrderService.cs:65]: Write this in form:
"try
            {
                string response = new ClientScheduledService().SendMessage(CreateOrder(order)).Result;
                return true;
            }
            catch (Exception e) { return false; }"

File: IntegrationWithPharmacies/Services/UrgentOrderService.cs [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        {
+            String[] parts = medicine.Split("_");
+
+            return new UrgentMedicineOrder(parts[0], int.Parse(parts[1]), pharmaciesWithMedicine[0].Api, DateTime.Now.ToString("dd/MM/yyyy"));
+        }
+
+        public List<MedicineName> CheckMedicineAvailability(string medicine)
+        {
+            String foundAvailability = HttpService.FormMedicineAvailabilityRequest(medicine);
+            return MedicineAvailabilityTable.FormMedicineAvailability(foundAvailability);
```
Comment (trki293) [inline IntegrationWithPharmacies/Services/UrgentOrderService.cs:82]: Write this in form:
"return MedicineAvailabilityTable.FormMedicineAvailability(HttpService.FormMedicineAvailabilityRequest(medicine));"

**Review by trki293 — CHANGES_REQUESTED** [review]
Correct the indicated places in the code, in order to meet the principles of the Clean code.

**Review by trki293 — APPROVED** [review]
PR is approved. The principles of the Clean code have been followed.

**Pull request merged**